packages/fxa-settings/src/components/Settings/ConnectedServices/index.tsx

```diff
@@ -171,6 +172,9 @@ export const ConnectedServices = forwardRef<HTMLDivElement>((_, ref) => {
   const onSignOutClick = useCallback(
     (c: AttachedClient) => {
       setSelectedClient(c);
+
+      GleanMetrics.accountPref.deviceSignout();
```

Comment:
Fly by comment / just wondering, I know I could probably find this on the Glean sheet, but is there a separate event for when users need to confirm from the sync modal? Just want to make sure Kimmy/Ross know about it, since I'm adding an event `reason` for the account unlinking that also has a modal.

Reply:
Does not appear to be a separate event for that, https://docs.google.com/spreadsheets/d/1DgohA2JG8gvFOjk_8zUHOi6DSDSPzqCCwoNVvirDpkA/edit?gid=1073857271#gid=1073857271. I'll ask them what they want.